### DISCUSSION TITLE
FAM use with Adopted/Foster

### DISCUSSION THREAD

**Discussion by jkr-wrk** [FamilySearch/GEDCOM#548] [Q&A] [answered]
What would be the right construction when two parents adopt or foster a child?

With a biological child, this is the normal way a GEDCOM shows:

```
0 @I1@ INDI
1 RIN 1
1 NAME Vader/Persoon/
1 FAMS @F1@
0 @I2@ INDI
1 RIN 2
1 NAME Moeder/Partner/
1 FAMS @F1@
0 @I3@ INDI
1 RIN 3
1 NAME Adoptie oudste/Kind/
1 FAMC @F1@
2 PEDI BIRTH
0 @F1@ FAM
1 HUSB @I1@
1 WIFE @I2@
1 CHIL @I3@
1 MARR
2 TYPE partners
```

But when it comes to none biological children someone tells me that this is not correct.
In that case it should be constructed in the following structure

```
0 @I1@ INDI
1 RIN 1
1 NAME Vader/Persoon/
1 FAMS @F1@
1 FAMS @F2@
0 @I2@ INDI
1 RIN 2
1 NAME Moeder/Partner/
1 FAMS @F1@
1 FAMS @F3@
0 @I3@ INDI
1 RIN 3
1 NAME Adoptie oudste/Kind/
1 FAMC @F2@
1 FAMC @F3@
2 PEDI FOSTER
0 @F1@ FAM
1 HUSB @I1@
1 WIFE @I2@
1 MARR
2 TYPE partners
0 @F2@ FAM
1 HUSB @I1@
1 CHIL @I3@
0 @F3@ FAM
1 WIFE @I2@
1 CHIL @I3@
```

This looks like a very strange and redundant construction when only the TYPE of the PEDI changes from BIRTH to FOSTER.

Is it true the construction for BIRTH and FOSTER must differ this way?

Edit: Fixed the typo in the second gedcom code, pointed out by @Norwegian-Sardines 
Edit: Fixed: Calling HUSB Moeder and WIFE Vader can cause some confusion.

**Comment by Norwegian-Sardines** [comment, marked as answer]
First, you can not have two:
0 @F1@ FAM

As in your suggested solution.

Second, I’d be doing it like this!

```
0 @I1@ INDI
1 RIN 1
1 NAME Vader /Persoon/
1 FAMS @F1@
0 @I2@ INDI
1 RIN 2
1 NAME Moeder /Partner/
1 FAMS @F1@
0 @I3@ INDI
1 RIN 3
1 NAME Adoptie oudste /Kind/
1 FAMC @F1@
2 PEDI FOSTER
0 @F1@ FAM
1 HUSB @I1@
1 WIFE @I2@
1 CHIL @I3@
```

I would be interested in the reasoning behind having multiple family records!

**Comment by jkr-wrk** [comment]
Reasoning with the second example:
"Your example GEDCOM indicates in the FAM segment that it is a civil marriage with three children from that marriage, thus biological; the indication for child 1, that this is an adopted child, was incorrectly specified by the creator (who??) of this GEDCOM."

The creator would be me.

Later:
"There are two possibilities for the FAM record:
- Unique relationship parent-parent-child (or single parent) and possibly specified marriage form with biological children.
- Unique relationship parent-child with non-biological children."

Then:
"We follow this GEDCOM method as specified by the creators."

I looked at the gedcom.io page and found nothing about parent-parent-child should be biological. I did find something about polyamory marriages, that to me creates an impression this FAM record is not that strict. It is indicated that if you want to register Parent-Parent-Parent-Child you could add the Child as a CHIL record and add all the parents as ASSO records.
I would not pick that solution myself, but it is clear that the child in this case is joined to the parents it has clearly no biological relation with, and there don't need to be 6 FAM records to tie it all together.

So that's why I try to get this answered 'by the creators'.

**Comment by jkr-wrk** [comment]
To address your subtle differences between your preferred and my preferred versions:

I fixed the switch between 'moeder' and 'vader' in the original post. 😀

Indeed the program does not include enough spaces, but webtrees can fix that issue:
https://webtrees.net/faq/name-spaces/

And yes, it looks a bit strange to add a MARR when there is no marriage.
Better to register this?
1 EVEN
2 TYPE Partners
2 DATE 4 SEP 2024

**Comment by Norwegian-Sardines** [comment]
I’m so sorry, but I’m not sure what you are asking!

If the two adult individuals are not known to be married then I would not indicate they were married with a MARR tag in the FAM record, but still include them in the record because in the USA I think all foster parents must be married.

The family_record does not require that the adults HUSB/WIFE be married.

The family_Record does not require that the child be a biological child, it can support adoption as well, by either adult or both adults.

**Comment by fisharebest** [comment]
There is a fundamental flaw in the GEDCOM structure here.

GEDCOM requires that children are adopted or fostered by a **Family**.

Whereas in the real world (well, in my country at least!), children are adopted or fostered by either one or two **individuals**, who may or may not be married.

So, adoption and fostering should be `INDI`:`INDI`, not `INDI`:`FAM`.

**Comment by jkr-wrk** [comment]
My second question (about MARR) might divert from my main question, i'm sorry.

@Norwegian-Sardines you answered my question pretty much as I expected. But exporting from Aldfaer to Webtrees goes wrong because Webtrees expects a single FAM structure, where Aldfaer makes the complex structure indicated by my second example in the opening post.
Edit: **Aldfaer points to the makers of GEDCOM to indicate no change can be made. So the question is: Do the makers of GEDCOM agree?**

@fisharebest If adoption/foster is done by only one of the parents, connection to the FAM already present would be wrong. In this case creating a new FAM record that only exists of one Parent and all adoption/foster Children could be the solution. But creating 8 FAM records to connect 2 Foster children to 2 Parents seems overkill.

Children born within a marriage should also be able to be disconnected from one of the parents. So I see no difference here. The default is both parents are connected, but this could be bypassed.

I want the GEDCOM to look like @Norwegian-Sardines example.

And it does not go wrong in a sense that Webtrees does not register the family connection, but every report will show wrong family structures. And looking at a foster parent there is a family with it's parents, with it's partner, with it's first foster child, with it's second foster child etc.

And the second issue is that Aldfaer registers every Parent-Parent connection as a MARR. Which will show up in Webtrees as a Marriage. So I have to remove MARR tags to add EVEN tags, when the type is Partners.

**Reply by dthaler** [reply]
Discussion in GEDCOM Steering Committee:
* We marked @Norwegian-Sardines first response as the answer to the original question.

> exporting from Aldfaer to Webtrees goes wrong because Webtrees expects a single FAM structure, where Aldfaer makes the complex structure indicated by my second example in the opening post.

* For the question quoted immediately above, GEDCOM has been around for many decades and the original makers aren't around to ask, but GEDCOM does not guarantee that two implementations won't result in loss of information or other problems, when exporting from one and importing into the other.  It doesn't mean that either is necessarily "broken" with respect to the GEDCOM specification. This is an unfortunate reality that dates from the early GEDCOM specifications.  There are often multiple ways to represent the same information in GEDCOM.  Sometimes the specification and/or [FAQ](https://gedcom.io/techfaqs/) gives recommendations about best practices among them.

> Edit: **Aldfaer points to the makers of GEDCOM to indicate no change can be made. So the question is: Do the makers of GEDCOM agree?**

If a change is requested that violates the GEDCOM specification, then it cannot be made while still following that specification. So changing from `PEDI FOSTER` to a new `FOSTER` structure would make it non-compliant with the specification. But most of the other ideas proposed here (one FAM, two FAM, etc.) are permitted by the current spec and an application would ideally support all of them when importing a GEDCOM file.

**Comment by Norwegian-Sardines** [comment]
>But exporting from Aldfaer to Webtrees goes wrong because Webtrees expects a single FAM structure, where Aldfaer makes the complex structure indicated by my second example in the opening post.
Edit: Aldfaer points to the makers of GEDCOM to indicate no change can be made. So the question is: Do the makers of GEDCOM agree?

Aldfaer would be wrong if they allow two FAM records with the same XREF!  The XREF is a key and must be unique!

**Reply by jkr-wrk** [reply]
I guess you refer to @F1@ in the gedcom twice. That was a typo I made and fixed.

**Comment by Norwegian-Sardines** [comment]
> If adoption/foster is done by only one of the parents, connection to the FAM already present would be wrong. In this case creating a new FAM record that only exists of one Parent and all adoption/foster Children could be the solution.

Adoption and Foster are a little different in implementation.  GEDCOM provides for an ADOP tag but not a FOSTer tag.  In the GEDCOM ADOP tag you can identify which parent or both Adopted the child.  Here is the code for the father adoption:

```
1 ADOP 
2 FAMC @F1@
3 ADOP HUSB
```

**Reply by jkr-wrk** [reply]
Using that would make it even more complex.

When biological, then single FAM record. When adopted, then ADOP record pointing to a FAM record (That could be a FAM record with a single INDI?) When fostered, then 3 FAM records, Father+Mother, Father+Child, Mother+Child)

Where it all can be registered by just using PEDI FOSTER, PEDI BIRTH, PEDI ADOPTED on the INDI record. And when only the mother adopts the child, make a new family. Because apparently, there is a new family because the man in the household has no saying over the child.

So my question remains, according to the 'makers' of GEDCOM, should I speak to the makers of Webtrees or Aldfaer to visualise the FOSTER family structure correct?

**Comment by Norwegian-Sardines** [comment]
I'm not sure any "maker" of GEDCOM would tell an application they are doing this wrong!

I believe the following would be correct and the way I have been doing it for 40 years!

```
0 @F1@ FAM
1 HUSB @I1@
1 WIFE @I2@
1 CHIL @I3@

0 @F2@ FAM
1 HUSB @I11@
1 WIFE @I2@
1 CHIL @I3@

0 @I1@ INDI
1 NAME Bio /Father/

0 @I2@ INDI
1 NAME Bio /Mother/

0 @I11@ INDI
1 NAME Adoptive /Father/

0 @I3@ INDI
1 NAME The /Child/
1 BIRT
2 DATE 11 JUN 1961
2 PLAC Idaho Falls, Bonneville, Idaho
2 FAMC @F1@ 
1 ADOP
2 DATE 16 MAR 1964
2 FAMC @F2@
3 ADOP HUSB
1 FAMC @F1@
2 PEDI birth
1 FAMC @F2@
2 PEDI adopted
```

NOTE 1: The Birth Family is @F1@
NOTE 2: The Adopted Family is @F2@
NOTE 3: The child points to both families using a FAMC with a PEDI indicating the way they are connected (birth or adoption) 
NOTE 4: The ADOP event tells us which parent adopted the child.
